### DISCUSSION TITLE
using pdfplumber with pdfrw

### DISCUSSION THREAD

**Discussion by zdenop** [jsvine/pdfplumber#352] [Ideas]
Is there any way, how to use pdfplumber with [pdfrw](https://github.com/pmaupin/pdfrw)? e.g. pass  `pdfrw.PdfReader("my_file.pdf").pages[0]` to `pdfplumber `?

My task is to modify (add barcode) a pdf file based on information in it.
At the moment I open file with pdfplumber and extract information.
Then I created barcode
Then I open once again the same file add the barcodes and with it to output.

**Comment by jsvine** [comment]
Hi @zdenop. I'm not very familiar with the internals of `pdfrw`, so I don't have an authoritative answer for you. But my general understanding is that `pdfrw` and `pdfminer.six`/`pdfplumber` represent in PDF information in very different ways, making it difficult for them to pass information back and forth.
